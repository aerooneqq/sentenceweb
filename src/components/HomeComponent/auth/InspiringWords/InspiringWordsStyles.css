@media screen and (min-width: 430px){ 
    .inspiringWord{
        font-family: Arial;
        text-align: left;
        color: orangered;
      }

        .inspiringWord:nth-child(1){
            font-size: 6em;
        }

        .inspiringWord:nth-child(2){
            font-size: 7em;
        }

        .inspiringWord:nth-child(3){
            font-size: 8em;
        }
}

@media screen and (max-width: 430px){ 
    .inspiringWord{
        font-family: Arial;
        text-align: left;
        color: orangered;
    
        text-align: center;
      }
      
        .inspiringWord:nth-child(1){
            font-size: 3em;
        }

        .inspiringWord:nth-child(2){
            font-size: 4em;
        }

        .inspiringWord:nth-child(3){
            font-size: 5em;
        }
}

@media screen and (min-width: 1824px){ 
    .inspiringWord{
        font-family: Arial;
        text-align: left;
        color: orangered;
      }
      
        .inspiringWord:nth-child(1){
            font-size: 6em;
        }

        .inspiringWord:nth-child(2){
            font-size: 7em;
        }
        
        .inspiringWord:nth-child(3){
            font-size: 8em;
        }
}