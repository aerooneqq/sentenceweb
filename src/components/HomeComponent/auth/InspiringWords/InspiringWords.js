import React, {Component} from "react";

//Styles
import "./InspiringWordsStyles.css";

export default class InspiringWords extends Component { 
     
    render() { 
        return ( 
            <div id="entrySentenceContainer">
                <div className="inspiringWord">Think.</div>
                <div className="inspiringWord">Create.</div>
                <div className="inspiringWord">Render.</div>
            </div>
        );
    }
}