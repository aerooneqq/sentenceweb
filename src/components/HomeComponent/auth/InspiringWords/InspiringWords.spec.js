import React from 'react'
import { shallow } from 'enzyme'
import InspiringWords from './InspiringWords';

describe("Inspiring words", () => { 
    it("should render correctly", () => { 
        const inspiringWords = shallow(<InspiringWords />);

        expect(inspiringWords.find("div.inspiringWord").at(0).text()).toBe("Think.");
        expect(inspiringWords.find("div.inspiringWord").at(1).text()).toBe("Create.");
        expect(inspiringWords.find("div.inspiringWord").at(2).text()).toBe("Render.");
    }); 
})