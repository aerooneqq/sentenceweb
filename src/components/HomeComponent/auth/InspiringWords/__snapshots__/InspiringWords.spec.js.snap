// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Inspiring words should render correctly 1`] = `null`;
